src/insert.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding utf-8 -*-
+"""Implementation of an insert sort."""
+
+import timeit
+
+def insert(input_list):
+    """The insertion sort method."""
+    if not isinstance(input_list, list):
+        raise TypeError
+    
+    ordered_list = []
+
+    for index in range(len(input_list)):
+        ordered_list.append(input_list[index])
+        idx = index
+        while idx > 0 and ordered_list[idx] < ordered_list[idx - 1]:
+            ordered_list[idx], ordered_list[idx - 1] = ordered_list[idx - 1], ordered_list[idx]
```

Comment:
This isn't quite the correct implementation for insertion sort; it's a little slower as you are doing more value assignment statements than necessary.

What you want to do instead is assign the "value to be re-inserted" to a variable, holding it inside as you iterate down the list (in your while loop) until you find the right spot to re-insert the value. As you are iterating down the list in your while loop, just shift values up (one assignment statement) instead of swapping them (two assignment statements).